corehq/motech/fhir/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from corehq.motech.fhir.views import get_view
+
+urlpatterns = [
+    url(r'^fhir/R4/(?P<resource_type>\w+)/(?P<resource_id>\w+)$', get_view, name="fhir_get_view"),
```

Comment:
You'll want to grab the FHIR version as well. You could hard-code it while we only support one version, but it would be better to get it from the URL. You can validate against `fhir.const.FHIR_VERSIONS`.

Reply:
Done